lib/ansible/modules/apt_key.py

```diff
@@ -274,13 +274,13 @@ def main():
             state=dict(type='str', default='present', choices=['absent', 'present']),
         ),
         supports_check_mode=True,
-        mutually_exclusive=(('data', 'filename', 'keyserver', 'url'),),
+        mutually_exclusive=(('data', 'file', 'keyserver', 'url'),),
     )
 
     key_id = module.params['id']
     url = module.params['url']
     data = module.params['data']
-    filename = module.params['file']
+    file = module.params['file']
```

Comment:
Do not change the variable name throughout the code. That is not necessary.

Reply:
Fixed in https://github.com/ansible/ansible/pull/70492/commits/06c2752a8881a39af9ca7b2e23c2c12c5bab188f.